### PR TITLE
Sor 3840  dockerize sortable prebid cache

### PR DESCRIPTION
**Pull request by a-drees**

### CODE REVIEW

File: backends/azure_table.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func NewAzureBackend(account string, key string) *AzureTableBackend {
 				buffer[1] = '"'
 				buffer[6] = '"'
 				buffer[7] = ']'
-				return buffer
+				return &buffer
```
Comment (marks-sortable) [inline backends/azure_table.go]: Are all these Golang changes ones we're merging from upstream?

File: Gopkg.toml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 
 [[constraint]]
-  name = "github.com/Sirupsen/logrus"
+  name = "github.com/sirupsen/logrus"
```
Comment (marks-sortable) [inline Gopkg.toml]: [Cool URIs don't change!](https://www.w3.org/Provider/Style/URI)

File: backends/azure_table.go (hunk 1 above)
Comment (a-drees) [reply backends/azure_table.go]: Yes they are

File: Gopkg.toml (hunk 2 above)
Comment (a-drees) [reply Gopkg.toml]: https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276

File: Dockerfile [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
-RUN apt-get install --assume-yes apt-utils
-RUN apt-get install -y ca-certificates
+RUN apt update
+RUN apt install --assume-yes apt-utils
+RUN apt install -y ca-certificates
 
-ADD ./prebid-cache /app/prebid-cache
+RUN mkdir /app
+COPY --from=builder /go/src/github.com/prebid/prebid-cache/prebid-cache /app/prebid-cache
+COPY --from=loadconfig loadconfig /loadconfig
```
Comment (marks-sortable) [inline Dockerfile]: You don't actually seem to be using this

**Comment by a-drees** [comment]
You're right, I'm not.

I figured I would just put it in to keep it in line with other docker images.

Keep it/ditch it?

- Alan

**Comment by marks-sortable** [comment]
I'd say ditch it. It's easy enough to re-add later if we start relying on secrets or SSL certs.

**Pull request merged**